python/lsst/daf/butler/core/config.py

```diff
@@ -769,6 +769,9 @@ def overrideParameters(configType, config, full, toUpdate=None, toCopy=None):
         toCopy : `tuple`, optional
             `tuple` of keys whose values should be copied from ``full``
             into ``config``.
+        overwrite : `bool`, optional
```

Comment:
I think it's a bit confusing for method named `overrideParameters` to take an argument which says "don't override".  Maybe rename it to `updateParameters` if it's not too much work? (Or maybe it's only confusing to me, I have not looked at this code much?)

Reply:
Yes, I was worrying about that too.  updateParameters sounds better.